package_manifest.yaml

```diff
@@ -1111,3 +1111,67 @@ pulpcore_packages:
     python-whitenoise: {}
     python-yarl: {}
     python-zipp: {}
+
+foreman_release_packages:
```

Comment:
This will need to be added up top as well: https://github.com/theforeman/foreman-packaging/blob/rpm/develop/package_manifest.yaml#L11

Reply:
Added, thanks!